### PR TITLE
1) registration style and authorization style are changed

### PR DESCRIPTION
**Pull request by vakhonin**
2) MessageUtil.class is removed

### CODE REVIEW

File: src/main/java/com/epam/magazinestore/controller/AccountController.java [hunk 1]
```diff
@@ -100,21 +111,21 @@ public ModelAndView login(ModelAndView modelAndView, HttpSession session) {
...
 
         if (accountService.isUserNameExists(name)) {
-            String message = UNSUCCESSFUL_REGISTRATION;
-            modelAndView.addObject(MESSAGE, message);
-            modelAndView.setViewName(REGISTRATION_JSP);
+            String errorMessage = messageSource.getMessage(ERROR_REGISTRATION, null, locale);
+            session.setAttribute(MESSAGE, errorMessage);
+            modelAndView.setViewName("redirect:" + REGISTRATION);
         } else {
             Account account = new Account();
```
Comment (AntonKiselyov) [inline src/main/java/com/epam/magazinestore/controller/AccountController.java]: What about builder ?

Also controllers mustn't contain any business logic!

File: src/main/java/com/epam/magazinestore/controller/AccountController.java [hunk 2]
```diff
@@ -135,7 +146,7 @@ public ModelAndView user(HttpSession session) {
         return mav;
     }
 
-    @GetMapping(value = "/account/subscriptions", produces = "application/json")
+    @GetMapping(value = ACCOUNT + SUBSCRIPTIONS, produces = "application/json")
```
Comment (AntonKiselyov) [inline src/main/java/com/epam/magazinestore/controller/AccountController.java]: MediaType.APPLICATIONJSONVALUE

File: src/main/java/com/epam/magazinestore/util/PathUtils.java [hunk 3]
```diff
@@ -16,6 +16,7 @@
     public static final String REGISTRATION = "/registration";
     public static final String SIGN_OUT = "/sign_out";
     public static final String SUBSCRIBE = "/subscribe";
+    public static final String SUBSCRIPTIONS = "/subscriptions";
     public static final String UNSUBSCRIBE = "/unsubscribe";
```
Comment (AntonKiselyov) [inline src/main/java/com/epam/magazinestore/util/PathUtils.java]: What is "unsubscribe" related to ?

File: src/main/java/com/epam/magazinestore/util/PathUtils.java [hunk 4]
```diff
@@ -16,6 +16,7 @@
...
     public static final String USER = "/user";
     public static final String PAID_PAYMENTS = "/paid_payments";
```
Comment (AntonKiselyov) [inline src/main/java/com/epam/magazinestore/util/PathUtils.java]: /payments/paid - you can

File: src/main/java/com/epam/magazinestore/controller/AccountController.java [hunk 5]
```diff
@@ -50,10 +47,18 @@
 
     private static final String MESSAGE = "message";
 
+    private static final String SUCCESS = "success";
+
     private static final String NAME = "account_name";
 
     private static final String FORMAT = "dd-MM-yyyy";
```
Comment (Jebsuz) [inline src/main/java/com/epam/magazinestore/controller/AccountController.java]: that does not tell anything to anyone. A `FORMAT` of what? Is it currency format? Maybe file format? hmmm.. maybe it is a command line command format? name your fields and variables meaningful so people would have a clue what it holds

**Pull request merged**